Frameworks/CoreGraphics/CGGradient.mm

```diff
@@ -36,7 +36,7 @@ - (void)dealloc {
 @end
 
 __CGGradient::__CGGradient() : _components(NULL), _locations(NULL) {
```

Comment:
#1541 will in the future

---
In reply to: [92309043](https://github.com/Microsoft/WinObjC/pull/1535#discussion_r92309043) [](ancestors = 92309043)